docs/content/python/new-roles/_index.md

```diff
@@ -4,12 +4,14 @@ weight: 1
 no_nav: true
 ---
 
-SQLAlchemy Roles Early Access
-=============================
-
-We're working on the next major version of the SQLAlchemy roles library.
-This library makes it easy to model role based access control in Polar
-and enforce it over your SQLAlchemy models.
+Roles for SQLAlchemy Early Access
+=================================
+
+We're working on the next major version of support for roles in Oso. Our
+first release is in the SQLAlchemy library. This library makes it easy
+to model role based access control in Polar and enforce it over your
+SQLAlchemy models. If you're interested in using Roles with other
```

Comment:
There are a few places where you use lowercase `roles` and a few where you capitalize it as `Roles`. We should pick one.

Reply:
You are right. Will update to lower case once I finish the wording.